experimentation/revolve2/experimentation/genotypes/cppnwin/modular_robot/v1/_body_develop.py

```diff
@@ -126,7 +127,7 @@ def __add_child(
         return None
     up = __rotate(module.up, forward, orientation)
 
-    child = child_type(orientation * (math.pi / 2.0))
+    child = child_type(orientation * (math.pi / 2.0), 0)
```

Comment:
can you change this to use keyword arguments. i think it's easy to make mistakes in the order of arguments here because we don't have any type hints

Reply:
furthermore, i think this `,0` is mistake? i don't see it in the init methods of Brick1 etc